readme.md

```diff
@@ -6,19 +6,62 @@
 [![David Dependencies Status][david-icon]][david]
 [![David devDependencies Status][david-dev-icon]][david-dev]
 
-A short description of what this project is about and it's core features. This might be followed by some points to clarify:
-- Do something with this
-- Supports other stuff
+A library that helps with updating [orders](https://dev.commercetools.com/http-api-projects-orders.html) into the [commercetools] platform.
 
 ## Usage
-How to use this project, possibly with a small code example. Could be split into CLI and direct usage.
+
 ### CLI
-...
-### JS
-...
+
+You can use the orders update from the command line using [`sphere-node-cli`](https://github.com/sphereio/sphere-node-cli).
+In order for the CLI to update orders, the file to update from must be JSON and follow the this structure:
+```
+{
+  "orders": [
+    <order>,
+    <order>,
+    ...
+  ]
+}
+```
+Then you can use this file using the cli:
+```
+sphere-node-cli -t order -p my-project-key -f ./orders.json
+```
+
+### Direct usage
+
+If you want more control, you can also use this library directly in JavaScript. To do this you first need to install it:
+```
+npm install @commercetools/orders-update --save-dev
```

Comment:
Yeah that doesn't make any sense. I blame copying from https://github.com/sphereio/sphere-product-type-import/#direct-usage
🍐